webapp/components/Notification/Notification.vue

```diff
@@ -1,40 +1,90 @@
 <template>
-  <ds-space :class="{ read: notification.read, notification: true }" margin-bottom="x-small">
+  <ds-space :class="{ read: notificationData.read, notification: true }" margin-bottom="x-small">
     <client-only>
       <ds-space margin-bottom="x-small">
-        <user-teaser :user="from.author" :date-time="from.createdAt" />
+        <user-teaser :user="notificationData.triggerer" :date-time="notificationData.createdAt" />
       </ds-space>
       <ds-text class="reason-text-for-test" color="soft">
-        {{ $t(`notifications.reason.${notification.reason}`) }}
+        <base-icon
+          v-if="notificationData.report"
+          name="balance-scale"
+          v-tooltip="{ content: $t('notifications.report.name'), placement: 'right' }"
+        />
+        <base-icon
+          v-else-if="notificationData.comment"
+          name="comment"
+          v-tooltip="{ content: $t('notifications.comment'), placement: 'right' }"
+        />
+        <base-icon
+          v-else
+          name="bookmark"
+          v-tooltip="{ content: $t('notifications.post'), placement: 'right' }"
+        />
+        {{
+          $t(`notifications.reason.${notificationData.reason}` + notificationData.reasonExtention)
+        }}
       </ds-text>
     </client-only>
     <ds-space margin-bottom="x-small" />
     <nuxt-link
-      class="notification-mention-post"
-      :to="{ name: 'post-id-slug', params, ...hashParam }"
+      class="notification-link-for-test"
```

Comment:
yeah, so if this is only used for testing, I would say remove it... 
favor `data-test`, which are removed in production
for cypress, we use a production image, so you wouldn't be able to use `data-test`
for this I think it would be sufficient to target `.notifcations-card`, which presumably is used for styling and is inside `nuxt-link` so should have the desired effect in `e2e` tests at least...

Reply:
Solved 👍🏼